i18n/en.json

```diff
@@ -1508,6 +1508,10 @@
     "id": "api.license.request_trial_license.app_error",
     "translation": "Unable to get a trial license, please try again or contact with support@mattermost.com."
   },
+  {
+    "id": "api.license.request_trial_license.no-site-url.app_error",
+    "translation": "Unable to request a trial license, please configure your SiteURL first in the web server section of the mattermost configuration."
```

Comment:
```suggestion
    "translation": "Unable to request a trial license. Please configure a Site URL in the web server section of the Mattermost System Console."
```

Reply:
Thanks, @jespino. Just a few changes. I'm 3/5 on "your Site URL" and "a Site URL". Which do you think works better?

Reply:
I think both works pretty well. by the way, this message is only going to be seen as part of the API response, because some design changes, the error is not shown in the user interface so... I going to pick the `a Site URL` option.